CONTRIBUTING.md

```diff
@@ -13,8 +13,8 @@ Truth assertions and be added to `<project>/src/test/java`.
 
 Please make sure your code compiles by running `mvn clean verify` which will
 execute both unit and integration test phases.  Additionally, consider using
-http://travis-ci.org to validate your branches before you even put them into
-pull requests.  All pull requests will be validated by Travis-ci in any case
+https://travis-ci.com to validate your branches before you even put them into
+pull requests.  All pull requests will be validated by Travis CI in any case
 and must pass before being merged.
```

Comment:
This needs to be reworked as the project has switched from Travis CI to GitHub Actions.